lib/scrape.js

```diff
@@ -232,25 +312,39 @@ async function findGitHubUser(displayName, org) {
 
   const shortName = validUsername(displayName)
 
-  const username = await findGitHubUserInOrg(displayName, org)
-  if (username) return username
+  const userFromRepo = await getGitHubUserFromRepoInfo(
+    org,
+    displayName,
+    shortName
+  )
+  if (userFromRepo) return userFromRepo
```

Comment:
so I could create account https://github.com/WinningIt , fork a Ubuntu repo, and my fake account gets linked?